src/components/tx/TxMarkers.js

```diff
@@ -0,0 +1,34 @@
+import React, { useState, useEffect } from 'react';
+import helpers from '../../utils/helpers';
+
+
+const TxMarkers = (props) => {
+  const [tx, setTx] = useState(props.tx);
+
+  useEffect(() => {
+    setTx({genesis: helpers.isGenesisBlock(props.tx.hash) || helpers.isGenesisTx(props.tx.hash)});
+  }, [props.tx]);
+
+  const genesisMarker = () => {
+    if (!tx.genesis) {
+      return null;
+    }
+
+    return (
+      <button className="info-hover-wrapper btn btn-link pl-0">
+        <i className="fa fa-certificate text-info" title="Genesis"></i>
```

Comment:
I don't like changing this to a badge. I think we should keep the "GENESIS" text and maybe change it to a badge so others can be added later.

Reply:
Changed back to text, updated the screenshots